jetty-security/src/main/java/org/eclipse/jetty/security/SecurityHandler.java

```diff
@@ -58,20 +61,34 @@
 public abstract class SecurityHandler extends HandlerWrapper implements Authenticator.AuthConfiguration
 {
     private static final Logger LOG = Log.getLogger(SecurityHandler.class);
+    private static final List<Authenticator.Factory> _authenticatorFactories = new ArrayList<>();
```

Comment:
in this style private statics start with `__` not `_`

Reply:
It should also be called `__knownAuthenticationFactories` or `__discoveredA..F...`